src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService regServ;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        regServ = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setLogin("UserLogin");
+        user.setPassword("Pass123");
+        user.setAge(25);
+    }
+
+    @Test
+    void register_null_ThrowException_Ok() {
+        try {
+            regServ.register(null);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown if User is null");
```

Comment:
if you don't check error message or something else from exception, you should  use `assertThrows`